pkg/api/register.go

```diff
@@ -43,6 +45,8 @@ var (
 func init() {
 	servicecataloginstall.Install(groupFactoryRegistry, Registry, Scheme)
 	settingsinstall.Install(groupFactoryRegistry, Registry, Scheme)
+	admissioninstall.Install(groupFactoryRegistry, Registry, Scheme)
```

Comment:
Given that this is not specific to Service Catalog (or Kubernetes API server), we should eventually move this generic admission-related code from the main `k8s.io/kubernetes` repo to `k8s.io/apiserver` and reuse it.
But since Service Catalog doesn't have a dependency on `k8s.io/kubernetes` anymore, it should be fine to copy the code into `pkg/kubernetes/...` for now.